docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to form 
+arugments about the proving 
+scheme. 
+
+These arguments are used to 
+check samples of elements, 
+against the original ones. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires and 
+allow us to express circuits
```

Comment:
Can you elaborate on how the permutation polynomial allows "us to express circuits with less differing vectors per polynomial. " . I'm not sure I understand what this means